mc-mark-more.el

```diff
@@ -29,6 +29,9 @@
 
 (require 'multiple-cursors-core)
 (require 'thingatpt)
+(require 'sgml-mode)
+
+(defvar mc/mark-more-like-this-extended-keymap (make-sparse-keymap))
```

Comment:
Not a huge fan of moving `mc/mark-more-like-this-extended-keymap` to the very top of the file. I would keep it grouped with the code that uses it.

Everything else looks very sensible though, good job!

Reply:
I moved it to above the first function which references it :slightly_smiling_face: